projects/lf-documentation/src/app/app.component.ts

```diff
@@ -12,7 +12,14 @@ interface ComponentNode {
 
 const TREE_DATA: ComponentNode[] = [
   { name: 'Overview', routerLink: RouterLinks.OVERVIEW },
-  { name: 'Getting Started', routerLink: RouterLinks.GETTING_STARTED },
+  {
+    name: 'Getting Started',
+    routerLink: RouterLinks.GETTING_STARTED,
+    children: [
+      { name: 'Example via CDN', routerLink: RouterLinks.GETTING_STARTED_CDN },
```

Comment:
Should be plural `Examples Using CDN`, `Examples Using NPM`.
Even if there is only one NPM example for consistency